src/middleware/write/index.js

```diff
@@ -0,0 +1,32 @@
+import db from '../../services';
+
+export const run = (ctx, next, database) => {
+  const { body, params, method } = ctx;
+  const { table, id } = params;
+  let dispatch;
+
+  switch (method) {
+    case 'post':
+      dispatch = database.create(table, body);
```

Comment:
Koa's `ctx.body` needs to get overwritten by each database response. Something like this:
```
dispatch.create(table, body).then(record => ctx.body = record);
```